# Replace with the desired positive y min range from base_link
pmin_range_y: 0.5
# Replace with the desired positive y max range from base_link
pmax_range_y: 5.0
# Replace with the desired negative y min range from base_link
nmin_range_y: -0.5
# Replace with the desired negative y max range from base_link
nmax_range_y: -5.0